runtime/darwinia/src/lib.rs

```diff
@@ -419,7 +437,7 @@ sp_api::impl_runtime_apis! {
 
 	impl fp_rpc::EthereumRuntimeRPCApi<Block> for Runtime {
 		fn chain_id() -> u64 {
-			<Runtime as pallet_evm::Config>::ChainId::get()
+			<<Runtime as pallet_evm::Config>::ChainId as frame_support::traits::Get<u64>>::get()
```

Comment:
Is this necessary? Looks like the original way is more concise.

Reply:
The compiler can not deduce this type.